js/src/dapps/registry/Application/application.js

```diff
@@ -5,35 +5,59 @@ import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
 const muiTheme = getMuiTheme(lightBaseTheme);
 
 import CircularProgress from 'material-ui/CircularProgress';
-import Status from '../Status';
-import Lookup from '../Lookup';
+import styles from './application.css';
+import Accounts from '../accounts';
+import Lookup from '../lookup';
+import Register from '../register';
+import Events from '../events';
+import Status from '../status';
 
 export default class Application extends Component {
-  static childContextTypes = {
-    muiTheme: PropTypes.object
-  };
+  static childContextTypes = { muiTheme: PropTypes.object };
   getChildContext () {
     return { muiTheme };
   }
 
+  static propTypes = {
+    actions: PropTypes.object,
```

Comment:
Shouldn't all (most?) of those be required?
Shouldn't we check the internal shape also?